lib/config/schema.json

```diff
@@ -138,6 +138,9 @@
         },
         "lazyRoutePrefix": {
           "type": "string"
+        },
+        "poll": {
+          "type": [ "boolean", "number" ]
```

Comment:
I noticed that in https://github.com/angular/angular-cli/commit/a163186a703ab84926e80bad037422ce619231c9 the custom-typings.d.ts file defined `poll` as only being a `number`. I've updated all of my `poll` definitions to reflect this.